pkgs/top-level/python-packages.nix

```diff
@@ -3750,12 +3750,14 @@ in {
 
   crayons = callPackage ../development/python-modules/crayons{ };
 
-  django = self.django_lts;
+  django = self.django_3_1;
```

Comment:
It might be wiser to just keep this the same as the lts version, that way we will have fewer breakage I expect. Anyway, I think we need to choose what versions of the packages we focus on, those compatible with the latest django, or those compatible with lts django. I prefer the latter because I think it becomes a mess otherwise..

Reply:
Hi.

I really think it makes the maintenance easier if the “main” django version we support is the LTS one. I initially added the `django_lts` alias to make it clear that `django` is the current LTS and not the latest available version.

I think we could however introduce a `django_latest` or something like this if we wish to offer a more up to date alternative.

An alternative (that would require a fair bit of rework) would be to propose a django subset within the python package set, and instantiate such subset for multiple django versions. I am not sure if going this way is worth the effort but can give a try (later) if you think this could be beneficial.

Reply:
No, let's not add more attributes, and let's remove `lts`. `django` will then always point to `lts`. The latest version we can add like I did here, but without aliasing it.

Reply:
Updated this PR.